media_driver/linux/common/codec/ddi/media_ddi_encode_avc.cpp

```diff
@@ -2066,3 +2066,12 @@ uint32_t DdiEncodeAvc::getQMatrixBufferSize()
 {
     return sizeof(VAIQMatrixBufferH264);
 }
+
+void DdiEncodeAvc::ClearPicParams()
+{
+    uint8_t ppsIdx = ((PCODEC_AVC_ENCODE_SLICE_PARAMS)(m_encodeCtx->pSliceParams))->pic_parameter_set_id;
+    PCODEC_AVC_ENCODE_PIC_PARAMS  picParams = (PCODEC_AVC_ENCODE_PIC_PARAMS)m_encodeCtx->pPicParams + ppsIdx;
+
+    if (picParams != nullptr && picParams->pDeltaQp != nullptr)
+            MOS_FreeMemory(picParams->pDeltaQp);
```

Comment:
please add '{' and  '}' for 'if' even there's only sentence.

Reply:
Sure. I will refine it. Thanks

Reply:
Also please set picParams->pDeltaQp = nullptr.

Reply:
added.